polygeist/tools/cgeist/Test/Verification/vecaccess.c

```diff
@@ -21,3 +21,55 @@ int test_cons_idx(int_vec v) {
 int test_var_idx(int_vec v, unsigned i) {
   return v[i];
 }
+
+// CHECK-LABEL:   func.func @test_store(
+// CHECK-SAME:                          %[[VAL_0:.*]]: memref<?xvector<3xi32>>,
+// CHECK-SAME:                          %[[VAL_1:.*]]: i32,
+// CHECK-SAME:                          %[[VAL_2:.*]]: i32) -> i32 attributes
+// CHECK-NEXT:      %[[VAL_3:.*]] = affine.load %[[VAL_0]][0] : memref<?xvector<3xi32>>
+// CHECK-NEXT:      %[[VAL_4:.*]] = vector.insertelement %[[VAL_2]], %[[VAL_3]]{{\[}}%[[VAL_1]] : i32] : vector<3xi32>
+// CHECK-NEXT:      affine.store %[[VAL_4]], %[[VAL_0]][0] : memref<?xvector<3xi32>>
+// CHECK-NEXT:      return %[[VAL_2]] : i32
```

Comment:
I wonder why we are returning `el` directly here rather than returning  `(*v)[idx]`